docs/reference/channel.md

```diff
@@ -6,7 +6,7 @@
 
 ## empty
 
-The `channel.empty` factory method, by definition, creates a channel that doesn't emit any value.
+The `Channel.empty` factory method, by definition, creates a channel that doesn't emit any value.
```

Comment:
Can this be avoided

Reply:
This was just added for consistent syntax highlighting until we agree on which one to use across the board

Reply:
yeah, but it's not nice to change it continuously in the docs. It was `Channel` originally, then become `channel` and now `Channel` again. Think it should stay `channel` at least until there's no consensus

Reply:
🙏